CONTRIB.md

```diff
@@ -10,7 +10,7 @@ If your git history is not so clean, please do rewrite before you submit your PR
 
 To submit your changes for review you have to fork the repository, push your new branch there and then create a Pull Request with `OpenOversight:develop` as the target.
 
-Use [PULL_REQUEST_TEMPLATE.md](/PULL_REQUEST_TEMPLATE.md) to create the description for your PR! (The template should populate automatically when you go to open the pull request.)
+Use [pull_request_template.md](/.github/pull_request_template.md) to create the description for your PR! (The template should populate automatically when you go to open the pull request.)
```

Comment:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository#adding-a-pull-request-template#adding-a-pull-request-template

Reply:
Adding it to this folder also removes clutter from the base folder while alluding to the role that the `pull_request_template.md` file plays.

Reply:
This makes sense to me. Thanks!
One general note: I prefer keeping changes that have nothing to do with each other separate, in this case either a separate PR, or rebasing/squashing this PR into just two commits so they can submitted via merge.
This for example makes cherry-picking specific commits easier, and it also makes it less likely that things are changed without any comment why

Reply:
I can move this to a different PR so that everything is separate. One second!

Reply:
https://github.com/lucyparsons/OpenOversight/pull/912